### ISSUE TITLE
NuGet Package for Microsoft.Web.Administration needs updating

### ISSUE THREAD

**Issue by thephoton** [MicrosoftDocs/iis-docs#970]
The version of Microsoft.Web.Administration.dll in the NuGet package hasn't been updated since 2018, this makes it impossible to set newer SSLFlags.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: cef882d6-10fa-1f34-3942-2f4c4c5a657b
* Version Independent ID: 3552ef90-1171-037d-c1ad-53688203bab4
* Content: [Microsoft.Web.Administration Namespace](https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.administration?redirectedfrom=MSDN&view=iis-dotnet)
* Content Source: [iis-dotnet/xml/ns-Microsoft.Web.Administration.xml](https://github.com/MicrosoftDocs/iis-dotnet-sdk-docs-pr/blob/live/iis-dotnet/xml/ns-Microsoft.Web.Administration.xml)
* Product: **iis**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **soshir**

**Comment by lextm** [comment]
If you found a need to consume the NuGet package, you are likely on the wrong path. I wrote about it in [a blog post](https://halfblood.pro/whats-microsoft-web-administration-and-the-horrible-facts-you-should-know-b82f2c974da6), and in short you should only consume the copy from `%windir%\system32\inetsrv\Microsoft.Web.Administration.dll`.

**Comment by txwizard** [comment]
There is no such package present on my spiffy new Windows 11 Pro machine. Might that be so because some component, such as IIS, is not installed?

**Comment by lextm** [comment]
@txwizard Correct. Since the only usage of `Microsoft.Web.Administration` is to manage IIS, it is part of IIS and need to be installed along with IIS on a clean Windows machine.
